client/src/components/assessments/PeriodicAssessmentResults.js

```diff
@@ -245,7 +245,7 @@ export const PeriodicAssessmentsRows = ({
         <tr>
           {periods.map((period, index) => {
             const periodDisplay = periodToString(period)
-            const addAssessmentLabel = `Make a new ${entity?.toString()} assessment for ${periodDisplay}`
+            const addAssessmentLabel = `Make a new assessment for ${periodDisplay}`
```

Comment:
Perhaps something like `Add ${assessmentConfig.label || "a new assessment"} for ${periodDisplay}` here as well?